src/assets/fonts.ts

```diff
@@ -0,0 +1,40 @@
+import archivo400url from '@fontsource/archivo/files/archivo-latin-400-normal.woff2?url';
+import archivo600url from '@fontsource/archivo/files/archivo-latin-600-normal.woff2?url';
+// urls above only resolve corrently in production, so we use the following for local development:
+import '@fontsource/archivo/latin-400.css';
+import '@fontsource/archivo/latin-600.css';
+
+export type Font = {
+  family: string;
+  weight: number;
+  style: string;
+  woff2Url: string;
+}
+
+export const fontFamilyArchivo = 'Archivo, sans-serif';
+
+export const fonts: Font[] = [
+  {
+    family: 'Archivo',
+    weight: 400,
+    style: 'normal',
+    woff2Url: archivo400url,
+  },
+  {
+    family: 'Archivo',
+    weight: 600,
+    style: 'normal',
+    woff2Url: archivo600url,
+  },
+];
+
+export const getFontCss = (font: Font) => {
```

Comment:
Note: import raw (`import archivo400css from '@fontsource/archivo/latin-400.css?raw';`) is not an option as it results in font urls starting with `./files/...` that are not resolved. So instead we generate the font declaration ourselves with the url that is resolved correctly in production.

Note 2: I've left out `font-display: swap;` as we don't have a fallback yet that doesn't cause a layout shift. See to do on Fontaine and font swap.

Reply:
Using [fontaine](https://github.com/unjs/fontaine) seems a bit involved. Rather keep this PR small and merge this first, to add font swap as perf enhancement in a separate PR.